docs/topics/practices.rst

```diff
@@ -1,264 +1,264 @@
-.. _topics-practices:
```

Comment:
It looks like in this file you had the line-ending issue. It also affects the `spiders.rst` file.

Git usually handles this automatically, maybe there’s something you can do in your Git configuration to have Git send the changes with the expected line ending even if locally you use a different line ending.

Reply:
I fixed the line-ending of `spiders.rst`. Also, `practices.rst` has no changes (I don't know it was pulled) so don't worry about reviewing it.

Reply:
Be aware that both files still show up in the pull request with the line-ending change.